package.json

```diff
@@ -23,7 +23,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "body-parser": "^1.19.0",
-    "clinical-trial-matching-service": "github:mcode/clinical-trial-matching-service",
+    "clinical-trial-matching-service": "github:mcode/clinical-trial-matching-service#improve-clinicaltrialgov",
```

Comment:
Just a reminder that this will need to be changed back before we actually merge.